lib/services/android-project-service.ts

```diff
@@ -179,25 +184,25 @@ class AndroidProjectService implements IPlatformProjectService {
 		this.$logger.info("Copying %s", projDir);
 		shell.cp("-Rf", projDir, destDir);
 
-		var targetDir = path.join(destDir, path.basename(projDir));
+		let targetDir = path.join(destDir, path.basename(projDir));
 		// TODO: parametrize targetSdk
-		var targetSdk = "android-17";
+		let targetSdk = this.targetApi || "android-17";
```

Comment:
is it possible to have `this.targetApi` still unpopulated? Shouldn't we call this.getTarget (, just to make sure targetApi has correct value?
As far as I see this.targetApi is populated only in this.getTarget method:
https://github.com/NativeScript/nativescript-cli/blob/totev/mnc/lib/services/android-project-service.ts#L379